client/my-sites/promote-post-i2/components/search-bar/index.tsx

```diff
@@ -179,15 +238,35 @@ export default function SearchBar( props: Props ) {
 		} );
 	};
 
+	const onCampaignChangeOrderOption = ( sort: DropdownOption ) => {
+		const newOrder = {
+			orderBy: sort.value,
+			order:
+				sort.value === 'budget_high_low' || sort.value === 'impressions_high_low' ? 'desc' : 'asc',
+		};
+
+		setCampaignSortOption( newOrder );
+		handleSetSearch( {
```

Comment:
@sbarbosa  how can we change this?

Reply:
This might help you. This seems triggering correctly in local. I'm unsure if this order prop exists also outside the WooCommerce side, so you might want to add some flag there. 

Add this :
```
if ( searchOptions.order ) {
		searchQueryParams += `&order=${ searchOptions.order.order }&order_by=${ searchOptions.order.orderBy }`;
	}
``` 

in`use-promote-post-campaigns-query-paged.ts` 

I added it like this:
<img width="1142" alt="image" src="https://github.com/Automattic/wp-calypso/assets/43957544/5d23a586-3751-4e94-aa12-d8f3a176a3d3">

Reply:
Hi :) I have added this logic. We'll need to modify something in DSP to make this work, right?

Reply:
Hi @ernestoSk13, I checked, and I can't find the logic anywhere in the code. I tested it and do not see the change in the network tab.

I tested locally in my machine with the code Javier shared and it worked Ok. 
Maybe you have some uncommited changes in your local?

Reply:
hi @sbarbosa  ... yes, that was it... I commited but forgot to push :(

Reply:
it is in the remote branch now